app/services/hyrax/repository_fixity_check_service.rb

```diff
@@ -1,6 +1,5 @@
 module Hyrax
   class RepositoryFixityCheckService
-    # TODO: this never seems to be used. Remove?
     def self.fixity_check_everything
```

Comment:
Can you add a comment here: `This is service that folks can use in their own rake tasks, etc.`  otherwise, it's not really "documented" 😉